dataland-frontend/src/frameworks/BaseFrameworkRegistry.ts

```diff
@@ -0,0 +1,21 @@
+import { type BaseFrameworkDefinition } from "@/frameworks/FrameworkDefinition";
+import { FrameworkDefinitions } from "@/frameworks/BaseFrameworkRegistryImports";
+
+/**
+ * Retrieve the framework definition for the provided framework identifier if it exists
+ * It is critical that no files in the base-framework-definition dependency tree import .vue files
```

Comment:
Can someone please shortly explain this new setup to me?

Reply:
Answered in one of the previous questions.

Reply:
ok